src/features/emails/store.ts

```diff
@@ -1,23 +1,26 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
 import { EmailFrame } from './types';
 import { EmailStats } from './hooks/useEmailStats';
-import { ZetkinEmail } from 'utils/types/zetkin';
-import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import {
   RemoteItem,
   remoteItem,
   RemoteList,
   remoteList,
 } from 'utils/storeUtils';
+import { ZetkinEmail, ZetkinLink } from 'utils/types/zetkin';
 
 export interface EmailStoreSlice {
   emailList: RemoteList<ZetkinEmail>;
   frameList: RemoteList<EmailFrame>;
+  linkList: RemoteList<ZetkinLink>;
```

Comment:
I believe this should be a map with different lists for each email.

```suggestion
  linksByEmailId: Record<number, RemoteList<ZetkinLink>>;
```